src/components/DateInput.jsx

```diff
@@ -242,8 +242,10 @@ class DateInput extends React.PureComponent {
           placeholder={placeholder}
           autoComplete={autoComplete}
           disabled={disabled}
+          role="combobox"
           readOnly={typeof readOnly === 'boolean' ? readOnly : isTouch}
           required={required}
+          aria-controls={id}
```

Comment:
This should be the `id` for the datepicker instead of the input. Do we have the datepicker id in the component?